.github/workflows/publish.yml

```diff
@@ -4,22 +4,23 @@ on:
   release:
     types:
       - created
+      - published
```

Comment:
Do we want both `created` and `published` or just `published`? As far as I can tell, `created` will cause a run when a `draft` is saved. Do we want that behaviour?

Reply:
> I put in both in there because I wasn't sure that a new published release would trigger the published event

Based on my reading of the docs, it should run in that case.

From https://docs.github.com/en/github-ae@latest/developers/webhooks-and-events/webhooks/webhook-events-and-payloads#webhook-payload-object-32:
> `published`: a release, pre-release, or draft of a release is published

A new published release _should_ fall into the set of events that fall under a release that's published if I'm interpreting that right.